nyctaxi/taxidata/README.md

```diff
@@ -1,67 +1,100 @@
+| `--verbose` | Log data | `false` |
+| `--yellowDataUrl` | URL to get the yellow Taxi data | https://s3.amazonaws.com/nyc-tlc/trip+data/yellow_tripdata_2019-01.csv) |
+
+This table lists authentication parameters supported by the `taxidata` tool.
+
+| Parameter | Description |
+| --- | --- |
+| `--auth-plugin` | The plugin for plugin authentication |
+| `--auth-params` | The parameters for plugin authentication |
+| `--token` | The token for token authentication |
+| `--token-file` | The file with a token for token authentication |
+| `--tls-cert-file` | The file with a TLS certificate for TLS authentication |
+| `--tls-key-file` | The file with a TLS private key for TLS authentication |
+| `--oauth2-issuer` | The issuer endpoint for OAuth2 authentication |
+| `--oauth2-audience` | The audience identifier for OAuth2 authentication |
+| `--oauth2-key-file` | The file with client credentials for OAuth2 authentication |
+
+### Examples
+
+This example shows how to use the `taxidata` tool to ingest both green and yellow NYC Taxi data to topics `public/default/taxidata-green` and `public/default/taxidata-yellow` in StreamNative Cloud.
+
+1. Create a Pulsar cluster through [StreamNative Cloud Manager](https://console.streamnative.cloud/login). For details, see [Create Pulsar Cluster](https://docs.streamnative.io/cloud/stable/use/cluster#create-cluster-through-streamnative-cloud-manager).
+
+2. Create a service account with the super-admin permission for the cluster and download the key file of the service account to your local computer. For details, see [Work with service account](https://docs.streamnative.io/cloud/stable/managed-access/service-account#work-with-service-account-through-streamnative-cloud-manager).
+
+3. Create a Flink cluster. For details, see [Create Flink cluster](https://docs.streamnative.io/cloud/stable/compute/flink-sql.md#create-flink-cluster). The Flink cluster is associated with the Pulsar cluster.
+
+4. Connect to the Pulsar cluster through the OAuth2 authentication plugin.
+
+	```bash
+	taxidata \
+	--oauth2-issuer https://auth.streamnative.cloud  \
+	--oauth2-audience urn:sn:pulsar:pulsar-namespace-name:pulsar-instance-name \
+	--oauth2-key-file /absolute path/to/key/file.json \
+	--pulsarUrl BROKER_SERVICE_URL
+	```
+
+5. Load the NYC Taxi data to topics `public/default/greenTaxi` and `public/default/yellowTaxi` in StreamNative Cloud.
+
+	```bash
+	taxidata \
+	--topicNameGreen public/default/greenTaxi \
+	--topicNameYellow public/default/yellowTaxi \
+	--oauth2-issuer https://auth.streamnative.cloud  \
+	--oauth2-audience urn:sn:pulsar:pulsar-namespace-name:pulsar-instance-name \
+	--oauth2-key-file /absolute path/to/key/file.json \
+	--pulsarUrl BROKER_SERVICE_URL
+	```
+
+6. Submit one or more FLink SQL query jobs through the StreamNative Cloud Manager.
+
+   1. On the left pane of the StreamNative Cloud Manager, click **SQL**.
+
+   2. Select the Flink database (Flink cluster) and Flink catalog (Pulsar cluster).
```

Comment:
Need to bold "Flink database (Flink cluster)" and "Flink catalog (Pulsar cluster)"?

Reply:
no need, because is not an icon or a button name.